src/OpenLoco/Ui/ViewportInteraction.cpp

```diff
@@ -313,7 +313,7 @@ namespace OpenLoco::Ui::ViewportInteraction
         // station | town | thing
         auto res = getMapCoordinatesFromPos(tempX, tempY, 0b11111111111111111100111111111101);
```

Comment:
these are all in terms of your new flags.

Reply:
(You can just express them as a negation of three particular flags.)